### DISCUSSION TITLE
Upgrading from Fast.Components.FluentUI to FluentUI.AspNetCore.Components

### DISCUSSION THREAD

**Discussion by nmdschultz** [microsoft/fluentui-blazor#2699] [Q&A]
Is there any guide on how to do this?
We currently run Microsoft.Fast.Components.FluentUI version 3.7.8, but have since upgraded to .NET8.

**Comment by nmdschultz** [comment]
It might be as simple as deleting the old package and installing the newer one..

**Reply by vnbaaij** [reply]
Yes, just install the new package. You will then of course need to change your `usings`, `_Imports.razor`, etc. 

And there are some other changes. See https://www.fluentui-blazor.net/UpgradeGuide#breaking-changes-v4.0.0

**Reply by nmdschultz** [reply]
Wow, thanks for the quick reply!
Your link also contains the guide I was looking for. :)
